possel/application.py

```diff
@@ -82,17 +82,19 @@ def get_etc_file(filename):
 
 def get_arg_parser():
     arg_parser = argparse.ArgumentParser(description='Possel Server')
-    arg_parser.add_argument('-d', '--database', default='sqlite:///possel.db',
+    arg_parser.add_argument('-d', '--database',
                             help='sqlalchemy-style database url string. See '
                             'http://peewee.readthedocs.org/en/latest/peewee/playhouse.html#db-url '
                             'for specification.')
-    arg_parser.add_argument('-p', '--port', default=80,
+    arg_parser.add_argument('-p', '--port',
                             help='Port possel server will listen on')
-    arg_parser.add_argument('-b', '--bind-address', default='',
+    arg_parser.add_argument('-b', '--bind-address',
```

Comment:
This probably wants to be something under `/etc` except programmatically defined (e.g. something like `get_etc_file('config.yml')`), can leave that for a different PR.